myosuite/envs/myo/walk_v0.py

```diff
@@ -366,3 +365,311 @@ def _get_angle(self, names):
         Get the angles of a list of named joints.
         """
         return np.array([self.sim.data.qpos[self.sim.model.jnt_qposadr[self.sim.model.joint_name2id(name)]] for name in names])
+    
+class TerrainEnvV0(BaseV0):
```

Comment:
shouldnt this inherit from WalkEnv? you could remove many of the redefined functions